docs/reference/rest-api/security/create-roles.asciidoc

```diff
@@ -50,6 +50,9 @@ privilege or action.
 `cluster`:: (list) A list of cluster privileges. These privileges define the
 cluster level actions that users with this role are able to execute.
 
+`description`:: (string) Allows providing a brief description of the role.
+The maximum length is `1000` chars.
```

Comment:
Question: was this originally 2048?

Reply:
Yes. My initial proposal was to allow max 2048 chars, but during the review, we [decided to lower it](https://github.com/elastic/elasticsearch/pull/107088/files/6aa7c12e774f75c52dfb3e0ab717537503e1feb7#r1584445783).